crates/noirc_evaluator/src/ssa_refactor/ssa_builder/mod.rs

```diff
@@ -276,8 +312,12 @@ impl FunctionBuilder {
         self.terminate_block_with(TerminatorInstruction::Jmp { destination, arguments });
     }
 
-    /// Terminate the current block with a jmpif instruction to jmp with the given arguments
-    /// block with the given arguments.
+    /// Terminate the current block with a conditional jump instruction.
+    ///
+    /// Jump to the `then` block if the condition is true, else jump to
+    /// the `else` block.
+    ///
+    /// TODO: where are the block arguments being supplied?
```

Comment:
`jmpif` never has any block arguments. It is just something we've never needed and thus have never needed to implement. If we ever expand the mem2reg pass to change references across multiple blocks into block arguments automatically we'd need to add it then.